src/find-build.js

```diff
@@ -1,26 +1,21 @@
 const fetch = require('node-fetch')
 
-const base = 'https://circleci.com/api/v2'
+const base = 'https://circleci.com/api'
 const projectSlug = 'gh/DataBiosphere/terra-ui'
 const buildArtifactPath = 'build.tgz'
 
 const findBuild = async () => {
-  // This request returns 401 but sets a cookie that we need for the job artifacts endpoint.
-  // Why only that endpoint needs the cookie is unknown.
-  const profileResponse = await fetch('https://circleci.com/api/v1.1/me')
-  const cookie = profileResponse.headers.raw()['set-cookie']
-
-  const workflows = await fetch(`${base}/insights/${projectSlug}/workflows/build-deploy?branch=dev`).then(r => r.json()).then(o => o.items)
+  const workflows = await fetch(`${base}/v2/insights/${projectSlug}/workflows/build-deploy?branch=dev`).then(r => r.json()).then(o => o.items)
   const latestWorkflow = workflows.find(w => w.status === 'success')
 
-  const workflow = await fetch(`${base}/workflow/${latestWorkflow.id}`).then(r => r.json())
-  const pipeline = await fetch(`${base}/pipeline/${workflow.pipeline_id}`).then(r => r.json())
+  const workflow = await fetch(`${base}/v2/workflow/${latestWorkflow.id}`).then(r => r.json())
+  const pipeline = await fetch(`${base}/v2/pipeline/${workflow.pipeline_id}`).then(r => r.json())
   const revision = pipeline.vcs.revision
 
-  const workflowJobs = await fetch(`${base}/workflow/${latestWorkflow.id}/job`).then(r => r.json()).then(o => o.items)
+  const workflowJobs = await fetch(`${base}/v2/workflow/${latestWorkflow.id}/job`).then(r => r.json()).then(o => o.items)
   const buildJob = workflowJobs.find(j => j.name === 'build' && j.status === 'success')
 
-  const jobArtifacts = await fetch(`${base}/project/${projectSlug}/${buildJob.job_number}/artifacts`, { headers: { cookie }}).then(r => r.json()).then(o => o.items)
+  const jobArtifacts = await fetch(`${base}/v1.1/project/${projectSlug}/${buildJob.job_number}/artifacts`).then(r => r.json());
```

Comment:
maybe a comment here explaining the findings in this PR and why its different from the other requests using v2

Reply:
Added in https://github.com/DataBiosphere/saturn-ui-prod-deploy/pull/44/commits/2107c9613efe215ae557613a8e1db35edba3b036.